lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -153,6 +153,25 @@ def display_name
     "#{domain_name} (#{ref})"
   end
 
+  def destroy_via_queue(user = User.current_user)
```

Comment:
Prefer `destroy_queue` as the method name.  The pattern we've established is `some_operation` and `some_operation_queue` as the two related methods.  (Ignore `scan` because that one broke the rules, but it is old 👴 )